docs/en/install-upgrade/installing-stack-demo-secure.asciidoc

```diff
@@ -0,0 +1,894 @@
++
+["source","shell"]
+----
+sudo usr/share/elasticsearch/bin/elasticsearch-keystore remove xpack.security.http.ssl.keystore.secure_password
+----
+
+. From the {es} installation directory, add the updated HTTP keystore password, using the password you generated for this keystore:
++
+["source","shell"]
+----
+sudo usr/share/elasticsearch/bin/elasticsearch-keystore add xpack.security.http.ssl.keystore.secure_password
+----
+
+. Before restarting {es}, you need to update the permissions and ownership of all of the certificate files.
+
+.. From the `etc/elasticsearch/certs/` directory, change the files to be owned by the `root:elasticsearch` group:
++
+["source","shell"]
+----
+chown root:elasticsearch *
+----
+
+.. Set the files in `/etc/elasticsearch/certs` to have read and write permissions by the owner (`root`) and read permission by the `elastic` user:
++
+["source","shell"]
+----
+chmod 640 *
+----
+
+.. Change the `/etc/elasticsearch/certs` and `/etc/elasticsearch/certs/ca` directories to be executable by the owner:
++
+["source","shell"]
+----
+chmod 750 /etc/elasticsearch/certs
+chmod 750 /etc/elasticsearch/certs/ca
+----
+
+. Restart the {es} service:
++
+["source","shell"]
+----
+sudo systemctl start elasticsearch.service
+----
+
+. Run the status command to confirm that {es} is running:
++
+["source","shell"]
+----
+sudo systemctl status elasticsearch.service
+----
++
+In the event of any problems, you can also monitor the {es} logs for any issues by tailing the {es} log file:
++
+["source","shell"]
+----
+sudo tail -f /var/log/elasticsearch/elasticsearch-demo.log
+----
++
+A line in the log file like the following indicates that SSL has been properly configured:
++
+["source","shell"]
+----
+[2023-07-12T13:11:29,154][INFO ][o.e.x.s.Security         ] [es-ssl-test] Security is enabled
+----
+
+[discrete]
+[[install-stack-demo-secure-second-node]]
+== Step 4: Configure security on additional {es} nodes
+
+Now that the security is configured for the first {es} node, some steps need to be repeated on each additional {es} node.
+
+. To avoid filename collisions, on each additional {es} node rename the existing `http.p12` file in the `/etc/elasticsearch/certs/` directory:
++
+["source","shell"]
+----
+mv http.p12 http-old.p12
+----
+
+. Copy the CA and truststore files that you generated on the first {es} node so that they can be reused on all other nodes:
+
+* Copy the `/ca` directory (that contains `ca.crt` and `ca.key`) from `/etc/elasticsearch/certs/` on the first {es} node to the same path on all other {es} nodes.
+
+* Copy the `elastic-stack-ca.p12` file from `/etc/elasticsearch/certs/` to the `/etc/elasticsearch/certs/` directory on all other {es} nodes.
+
+* Copy the `http.p12` file from each node directory in `/etc/elasticsearch/certs/elasticsearch/` (that is, `elasticsearch/mynode-es1`, `elasticsearch/mynode-es2` and `elasticsearch/mynode-es3`) to the `/etc/elasticsearch/certs/` directory on each corresponding cluster node.
```

Comment:
Since we've made some adjustments to the docs in prior steps, we need to change the first reference to the directory path to be `/usr/share/elasticsearch/elasticsearch` rather than `/etc/elasticsearch/certs/elasticsearch`. Everything else is good.